doc/modules/calibration.rst

```diff
@@ -208,3 +208,124 @@ a similar decrease in log-loss.
     .. [5] On the combination of forecast probabilities for
            consecutive precipitation periods. Wea. Forecasting, 5, 640–650.,
            Wilks, D. S., 1990a
+
+
+.. _decision_threshold_calibration:
+
+==============================
+Decision Threshold calibration
+==============================
+
+.. currentmodule:: sklearn.calibration
+
+Often Machine Learning classifiers base their
+predictions on real-valued decision functions or probability estimates that
```

Comment:
"that" -> ". These"

Although I think we might land up rewriting some of this. I think these three paragraphs would be clearer with something like:
"Binary and multilabel classifiers often choose the class by thresholding a real-valued decision function. The default threshold may not be well calibrated for maximizing some specific evaluation metric. :class:`CutoffClassifier` can wrap a binary classifier to calibrate a task-appropriate decision threshold.
"